### PR TITLE
feat: improvements on glossary screen reader

### PR DESCRIPTION
**Pull request by PassionPenguin**
### Description

- added ChromeVox, Narrator description.
- links improvements

### CODE REVIEW

File: files/en-us/glossary/screen_reader/index.md [hunk 1]
```diff
@@ -10,32 +10,32 @@ Screen readers are software applications that attempt to convey what is seen on
...
+iOS and macOS comes with VoiceOver, a built-in screen reader. To access VoiceOver on macOS, go to System Settings > Accessibility > VoiceOver. You can also toggle VoiceOver on and off with fn + Command + F5. VoiceOver both reads aloud and displays content. The content read aloud is displayed in a dark grey box.
 
-Just like keyboard navigation without VoiceOver, you can navigate through interactive elements using the tab key and the arrow keys:
+See [VoiceOver User Guide for Mac](https://support.apple.com/guide/voiceover/welcome/mac) and [Use VoiceOver in apps on iPhone](https://support.apple.com/guide/iphone/use-voiceover-in-apps-iphe4ee74be8/ios) for more information.
 
-- Next interactive element: Tab
-- Previous interactive element: Shift + Tab
-- Next radio button in a same named-group: right or down arrow
-- Previous radio button in a same named-group: left or up arrow
+### Narator
```
Comment (estelle) [inline files/en-us/glossary/screen_reader/index.md:21]:
```suggestion
### Narrator
```

File: files/en-us/glossary/screen_reader/index.md [hunk 2]
```diff
@@ -10,32 +10,32 @@ Screen readers are software applications that attempt to convey what is seen on
 
 In terms of web accessibility, most user agents provide an accessibility object model and screen readers interact with dedicated accessibility APIs, using various operating system features and employing hooking techniques.
```
Comment (estelle) [inline files/en-us/glossary/screen_reader/index.md:11]:
```suggestion
In terms of web {{glossary("accessibility")}}, most user agents provide an accessibility object model and screen readers interact with dedicated accessibility APIs, using various operating system features and employing hooking techniques.
```

**Review by estelle — COMMENTED** [review]
I'm hesitant to get rid of voice over instructions, especially if we keep the instructions on how to turn it on. Any thoughts on where we could put the instructions? or a link to the instructions for all of them?

**Comment by PassionPenguin** [comment]
> I'm hesitant to get rid of voice over instructions, especially if we keep the instructions on how to turn it on. Any thoughts on where we could put the instructions? or a link to the instructions for all of them?

u're right. let's add a link to [VoiceOver general commands on Mac](https://support.apple.com/guide/voiceover/cpvokys01/mac)

File: files/en-us/glossary/screen_reader/index.md [hunk 3]
```diff
@@ -10,32 +10,32 @@ Screen readers are software applications that attempt to convey what is seen on
...
+## Common screen readers
 
-macOS comes with VoiceOver, a built-in screen reader. To access VoiceOver, go to System Preferences > Accessibility > VoiceOver. You can also toggle VoiceOver on and off with fn+command + F5. VoiceOver both reads aloud and displays content. The content read aloud is displayed in a dark grey box.
+### VoiceOver
 
-Desktop/laptop screen reader users navigate websites with a keyboard or other non-pointing device. The best way to emulate use is to do the same.
+iOS and macOS comes with VoiceOver, a built-in screen reader. To access VoiceOver on macOS, go to System Settings > Accessibility > VoiceOver. You can also toggle VoiceOver on and off with fn + Command + F5. VoiceOver both reads aloud and displays content. The content read aloud is displayed in a dark grey box.
 
-Just like keyboard navigation without VoiceOver, you can navigate through interactive elements using the tab key and the arrow keys:
+See [VoiceOver User Guide for Mac](https://support.apple.com/guide/voiceover/welcome/mac) and [Use VoiceOver in apps on iPhone](https://support.apple.com/guide/iphone/use-voiceover-in-apps-iphe4ee74be8/ios) for more information.
```
Comment (PassionPenguin) [inline files/en-us/glossary/screen_reader/index.md:19]:
```suggestion
VoiceOver can also be used with commands shortcuts. See [VoiceOver general commands on Mac](https://support.apple.com/en-sg/guide/voiceover/cpvokys01/mac) for a complete list.

See [VoiceOver User Guide for Mac](https://support.apple.com/guide/voiceover/welcome/mac) and [Use VoiceOver in apps on iPhone](https://support.apple.com/guide/iphone/use-voiceover-in-apps-iphe4ee74be8/ios) for more information.
```

**Review by estelle — APPROVED** [review]

**Pull request merged**